contracts/interchain-token/InterchainToken.sol

```diff
@@ -130,7 +130,6 @@ contract InterchainToken is InterchainTokenStandard, ERC20, ERC20Permit, Minter,
 
     /**
      * @notice A method to be overwritten that will decrease the allowance of the `spender` from `sender` by `amount`.
-     * @dev Needs to be overwritten. This provides flexibility for the choice of ERC20 implementation used. Must revert if allowance is not sufficient.
```

Comment:
nice to have but i'd rather not change the token bytecode unless needed to avoid two different token versions in production. So, we can close this PR/task for now